src/prognostic_equations/edmfx_closures.jl

```diff
@@ -229,3 +230,107 @@ function edmfx_entr_detr_tendency!(Yₜ, Y, p, t, colidx, turbconv_model::EDMFX)
     end
     return nothing
 end
+
+# lambert_2_over_e(::Type{FT}) where {FT} = FT(LambertW.lambertw(FT(2) / FT(MathConstants.e)))
```

Comment:
Why the bottom one is better?

Reply:
IIRC, `lambertw` is a pretty complicated function, and since we're only passing a constant in (`FT(2) / FT(MathConstants.e)`), we micro optimized this by precomputing it and sticking in the constant. It may have also been causing inference issues, which can percolate downstream.